# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/systemservices/android_video_capture/video_capture_android.h
# Commit: 381836e3bad0
# Full Hash: 381836e3bad0647b91fea45fe629fe388c5d2222
# Author: Dan Minor <dminor@mozilla.com>
# Date: 2020-02-11 21:47:36
# Regressor Bug: 1578073
# File Overlap Count: 1
# Description:
#   Bug 1578073 - Fix include paths for android video capture; r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D61857
# ==============================================================================

diff -r 0c214e8034bb -r 381836e3bad0 dom/media/systemservices/android_video_capture/video_capture_android.h
--- a/dom/media/systemservices/android_video_capture/video_capture_android.h	Mon Feb 10 17:11:01 2020 +0000
+++ b/dom/media/systemservices/android_video_capture/video_capture_android.h	Mon Feb 10 17:11:29 2020 +0000
@@ -13,7 +13,7 @@
 
 #include <jni.h>
 
-#include "modules/video_capture/android/device_info_android.h"
+#include "device_info_android.h"
 #include "modules/video_capture/video_capture_impl.h"
 
 namespace webrtc {
